charts/jenkins/README.md

```diff
@@ -20,9 +20,6 @@ _See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation
 ```console
 # Helm 3
```

Comment:
I don't think this is needed.

Reply:
You men the reference to Helm 3? I was unsure if we should keep it or not. It somehow makes it explicit that this is for helm 3 on the other hand helm v2 will be end of life in Novemeber.